docs/_includes/footer.html

```diff
@@ -7,9 +7,14 @@
         <div class="block block__sub">
             <small>
                 <strong>Page last edited:</strong>
-                <time datetime="{{ page.last_modified_at }}">
-                    {{ page.last_modified_at | date: "%l:%M:%S %p %b %d, %Y" }}
-                </time>
+                {% if page.section %}
```

Comment:
This conditional is here because not all pages (like the homepage) have a page section.

Reply:
So within this `if` block you assign `file_url_section`, but then you use it outside the `if` block on L:13. The `if` should enclose the link too, no? Or provide a non-link version?

Reply:
Yeah it only gets defined if needed. Otherwise, it's undefined and prints nothing on L13. I can change the logic if it's confusing.

Reply:
Maybe this?

```
{% assign file_url_path = page.path | split: '/' | last %}
{% if page.section %}
    {% assign file_url_path = page.section | append: '/' | concat: file_url_path %}
{% endif %}
<a href="https://github.com/{{ site.repository }}/commits/master/docs/{{ file_url_path }}">
```

Reply:
Works for me 👍